app/src/main/java/com/sun/doitpat/ui/home/ToDoSwipeAdapter.kt

```diff
@@ -10,11 +10,15 @@ import com.sun.doitpat.R
 import com.sun.doitpat.base.SwipeBindingViewHolder
 import com.sun.doitpat.data.model.ToDo
 import com.sun.doitpat.util.Constants.COMPLETED
+import com.sun.doitpat.util.Constants.EMPTY_STRING
 import kotlinx.android.synthetic.main.item_reminder_swipe.view.*
 
 class ToDoSwipeAdapter(private val listener: OnSwipeItem) : RecyclerSwipeAdapter<SwipeBindingViewHolder<ToDo>>() {
 
     private var items = mutableListOf<ToDo>()
+    private var filteredItems = mutableListOf<ToDo>()
+    private var filterItems = listOf<ToDo>()
```

Comment:
List này có vẻ hơi thừa nhỉ?

Reply:
1 list dùng để backup khi thay đổi kí tự search. 1 list chữa các item đã lọc. Còn items là list gốc để hiện thị a ạ

Reply:
Anh nghĩ không cần thiết lắm. Em có items để lấy toàn bộ data, filteredItems để hiển thị các item sau khi search thì cứ notify theo filteredItems thôi